lib/bitcoin/BitcoinProcessor.ts

```diff
@@ -82,6 +86,16 @@ export default class BitcoinProcessor {
 
   private serviceInfo: ServiceInfo;
 
+  /** proof of fee configuration */
+  private readonly proofOfFeeConfig: IProofOfFeeConfig;
+
+  private readonly quantileCalculator: SlidingWindowQuantileCalculator;
+
+  private readonly transactionSampler: ReservoirSampler;
+
+  /** satoshis per bitcoin */
+  private static readonly satoshiPerBTC = 100000000;
```

Comment:
Nit: casing, should be `satoshiPerBtc`, but would actually prefer `satoshisPerBitcoin`.

Reply:
Went with satoshisPerBitcoin